tests/scanner/audit/data/log_sink_test_valid_rules.yaml

```diff
@@ -67,3 +67,14 @@ rules:
       destination: '*'
       filter: ''
       include_children: '*'
+  - name: 'Only allow Billing Account sinks to audit logs project.'
+    mode: whitelist
+    resource:
+      - type: billing_account
+        applies_to: self
+        resource_ids:
+          - ABCD-1234
```

Comment:
nit: a more descriptive resource name may make it easier to understand the tests (e.g. FOO-BILLING, BILLING-ACCT-1, etc).

Reply:
I was keeping these looking somewhat like a real billing account, which are hexadecimal strings with hyphens, but longer: 012345-6789AB-CDEF01